wolfssl/wolfcrypt/types.h

```diff
@@ -125,7 +125,15 @@
         WOLFSSL_WORD_BITS  = WOLFSSL_WORD_SIZE * WOLFSSL_BIT_SIZE
     };
 
-    #define WOLFSSL_MAX_16BIT 0xffffU
+    #define WOLFSSL_MAX_16BIT  0xffffU
+
+#ifdef WOLFSSL_OTHER_MAX_TYPES
+    #define WOLFSSL_MAX_BYTE   0xff
+    #define WOLFSSL_MAX_32BIT (~((word32)0U))
```

Comment:
Why is U used for the type when sha256.c for example uses L for 32bits?

Reply:
Sorry for interference, this definition was my suggestion. The U means it's an unsigned numeric constant, which makes sense to define a maximum value of an unsigned type. It doesn't imply the size. L means a long int literal, which doesn't make sense when used with a 32bit type unless it's a 16bit machine. In case of sha256.c, there is no need to write the U as the literals are in hexa format.

Reply:
@kaleb-himes : Can we just make this `0xffffffffU` like the others?